playbooks/roles/memcached/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- name: install memcached
+  apt: pkg=memcached state=present
```

Comment:
@mrgnr @tkeemon Thanks, I wasn't aware of the edX ansible conventions. The handler is removed, and tasks are following the conventions now.